src/main/scala/com/modelfabric/sparql/api/SparqlConstruct.scala

```diff
@@ -12,8 +13,10 @@ object SparqlConstruct extends SparqlConstructFactory {
             valueIRIs: Seq[IRI] = Nil,
             graphIRIs: Seq[IRI] = Nil,
             reasoningEnabled: Boolean = false)(
+           // JC why make _paging implicit?
```

Comment:
historically it was always implicit... want to take on the refactoring task now to remove it?

Reply:
wanted to understand if there was any consideration behind it. implicit helps in some scenarios, but we also need to be careful. probably not right now. once we have more changes in this lib